napari/_qt/_qapp_model/_tests/test_processors.py

```diff
@@ -18,3 +51,18 @@ def test_add_layer_data_to_viewer():
             return_type=Union[ImageData, LabelsData],
             viewer=v,
         )
+
+
+def test_add_layer_to_viewer():
+    layer = MagicMock()
+    viewer = MagicMock()
+    _add_layer_to_viewer(None)
+    _add_layer_to_viewer(layer)
+    _add_layer_to_viewer(layer, viewer)
```

Comment:
similar to above could we make a viewer and check that the layer has been added?